python/gcp_client/LICENSE

```diff
@@ -0,0 +1,7 @@
+“Software code created by U.S. Government employees is not subject to copyright
```

Comment:
LICENSE can also probably be symlinked. It will be worth testing how `setuptools` plays with the symlink when you for example build a wheel.